boards/riscv/longan_nano/longan_nano.dts

```diff
@@ -7,99 +7,9 @@
 /dts-v1/;
 #include <gigadevice/gd32vf103Xb.dtsi>
 #include "longan_nano-pinctrl.dtsi"
+#include "longan_nano_common.dtsi"
```

Comment:
I was wondering if we should use `<board name>_foo.dtsi` or `<board name>-bar.dtsi`.

Reply:
I agree.  Correct it.